ApiBundle/Resources/translations/datatable.en.yml

```diff
@@ -1,6 +1,6 @@
 open_orchestra_datatable:
     empty_table: No data available in table
-    info: There are %total% results
+    info: "{0} There is %total% result|[1,Inf] There are %total% results"
```

Comment:
@alavieille : {0} There is no result, {1} There is one result, [2+] There are xxx results